homeassistant/components/litterrobot/sensor.py

```diff
@@ -118,7 +132,15 @@ async def async_setup_entry(
     """Set up Litter-Robot sensors using config entry."""
     hub: LitterRobotHub = hass.data[DOMAIN][entry.entry_id]
     async_add_entities(
-        LitterRobotSensorEntity(robot=robot, hub=hub, description=description)
-        for description in ROBOT_SENSORS
-        for robot in hub.litter_robots()
+        [
```

Comment:
You could probably wrap this in `itertools.chain` or `itertools.chain.from_iterable()` but it likely doesn't matter here.